src/components/FlatParameters/FlatParameters.js

```diff
@@ -0,0 +1,265 @@
+                                m,
+                                e.target.value
+                            );
+                            if (m) {
+                                onFieldChange(e.target.value, param);
+                            }
+                        }}
+                        variant={'standard'}
+                    />
+                );
+            case 'STRING_LIST':
+                if (param.possibleValues) {
+                    return (
+                        <Autocomplete
+                            fullWidth
+                            multiple
+                            options={param.possibleValues}
+                            onChange={(e, value) => onFieldChange(value, param)}
+                            value={value}
+                            renderTags={(value, getTagProps) =>
+                                value.map((option, index) => (
+                                    <Chip
+                                        variant="outlined"
+                                        label={option}
+                                        {...getTagProps({ index })}
+                                    />
+                                ))
+                            }
+                            renderInput={(options) => (
+                                <TextField {...options} variant="standard" />
+                            )}
+                        />
+                    );
+                }
+            // else fallthrough to default
+            case 'STRING':
+                if (param.possibleValues) {
+                    return (
+                        <Autocomplete
+                            fullWidth
+                            disableClearable
+                            options={param.possibleValues}
+                            onChange={(e, value) => onFieldChange(value, param)}
+                            value={value}
+                            renderInput={(options) => (
+                                <TextField {...options} variant="standard" />
+                            )}
+                        />
+                    );
+                }
+            // else fallthrough to default
+            default:
+                return (
+                    <TextField
+                        fullWidth
+                        defaultValue={value}
+                        onFocus={() => onUncommited(param, true)}
+                        onBlur={() => onUncommited(param, false)}
+                        onChange={(e) => onFieldChange(e.target.value, param)}
+                        variant={'standard'}
+                    />
+                );
+        }
+    };
+
+    const renderAsList = () => {
+        return (
+            <List className={classes.paramList}>
+                {paramsAsArray.map((param) => (
+                    <Tooltip
+                        title={param.description}
+                        enterDelay={1200}
```

Comment:
Magic number, use a constant somewhere.
see `TOOLTIP_DELAY` in study-app which is 1000 by the way.
This constant should be in commons-ui ?

Reply:
May be. I just moved that code from useImportExport

Reply:
will see later then